### ISSUE TITLE
Transaction.Response - Both generatedKeys() and getGeneratedKeys() are marked as deprecated

### ISSUE THREAD

**Issue by sai-pullabhotla** [GoogleCloudPlatform/google-cloud-java#1351]
In the Datastore API 0.5.0, in Transaction.Response interface - Both generatedKeys() and getGeneratedKeys() are marked as deprecated. What is the alternative? I wonder if the new method getGeneratedKeys was unintentionally marked as deprecated? 

```
public interface Transaction extends DatastoreBatchWriter, DatastoreReaderWriter {

  interface Response {
    /**
     * Returns a list of keys generated by a transaction.
     */
    @Deprecated
    List<Key> generatedKeys();

    /**
     * Returns a list of keys generated by a transaction.
     */
    @Deprecated
    List<Key> getGeneratedKeys();
  }
//...
}

```

**Comment by mziccard** [comment]
@sai-pullabhotla Thanks for this report. This is clearly an error on our side, `getGeneratedKeys` should not be deprecated. I'll fix this ASAP.

**Comment by mziccard** [comment]
Fixed with #1358
